apps/console/src/features/organizations/pages/organization-roles.tsx

```diff
@@ -0,0 +1,492 @@
+     *
+     * @param {number} limit - List limit.
+     * @param {number} offset - List offset.
+     * @param {string} filter - Search query.
+     * @param {string} after - After link for cursor based pagination
+     * @param {string} before - Before link for cursor based pagination
+     */
+    const getOrganizationRoleLists = useCallback(
+        (limit?: number, filter?: string, after?: string, before?: string): void => {
+            setOrganizationRoleListRequestLoading(true);
+
+            getOrganizationRoles(
+                currentOrganization.id,
+                filter,
+                limit,
+                after,
+                before)
+                .then((response: OrganizationRoleListResponseInterface) => {
+                    handleNextButtonVisibility(response);
+                    setOrganizationRoles(response.roles);
+                })
+                .catch((error) => {
+                    if (error?.description) {
+                        dispatch(
+                            addAlert({
+                                description: error.description,
+                                level: AlertLevels.ERROR,
+                                message: t(
+                                    "console:manage.features.organizations.notifications." + // ToDo
+                                    "getOrganizationList.error.message"
+                                )
+                            })
+                        );
+
+                        return;
+                    }
+
+                    dispatch(
+                        addAlert({
+                            description: t(
+                                "console:manage.features.organizations.notifications.getOrganizationList" + // ToDo
+                                ".genericError.description"
+                            ),
+                            level: AlertLevels.ERROR,
+                            message: t(
+                                "console:manage.features.organizations.notifications." + // ToDo
+                                "getOrganizationList.genericError.message"
+                            )
+                        })
+                    );
+                })
+                .finally(() => {
+                    setOrganizationRoleListRequestLoading(false);
+                    setLoading(false);
+                });
+        },
+        [ dispatch, t ]
+    );
+
+    /**
+     * Sets the list sorting strategy.
+     *
+     * @param {React.SyntheticEvent<HTMLElement>} event - The event.
+     * @param {DropdownProps} data - Dropdown data.
+     */
+    const handleListSortingStrategyOnChange = (event: SyntheticEvent<HTMLElement>, data: DropdownProps): void => {
+        setListSortingStrategy(
+            find(ORGANIZATION_ROLES_LIST_SORTING_OPTIONS, (option) => {
+                return data.value === option.value;
+            })
+        );
+    };
+
+    /**
+     *
```

Comment:
Remove unnecessary line

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/81488c55dd6c9c7a63575dee78e71f2b80643963